e2e/validation_test.go

```diff
@@ -7,18 +7,22 @@ import (
 	"testing"
 
 	"github.com/ramendr/ramen/e2e/test"
-	"github.com/ramendr/ramen/e2e/util"
+	"github.com/ramendr/ramen/e2e/validate"
 )
 
 func TestValidation(dt *testing.T) {
 	t := test.WithLog(dt, Ctx.log)
 	t.Parallel()
 
+	if err := validate.TestConfig(Ctx.env, Ctx.config, Ctx.log); err != nil {
+		t.Fatalf("Failed to validate test environment with DR configurations: %s", err)
+	}
+
 	t.Run("hub", func(dt *testing.T) {
 		t := test.WithLog(dt, Ctx.log)
 		t.Parallel()
 
-		err := util.ValidateRamenHubOperator(Ctx.env.Hub, Ctx.config, Ctx.log)
+		err := validate.RamenHubOperator(Ctx.env.Hub, Ctx.config, Ctx.log)
 		if err != nil {
 			t.Fatalf("Failed to validate hub cluster %q: %s", Ctx.env.Hub.Name, err)
```

Comment:
For later: add a good error in validate.*Operator so we don't need to repeat the error messages here, and we can reuse the error messages in `ramenctl validate clusters` command.

Reply:
ack, created issue #1952